.rubocop.yml

```diff
@@ -373,7 +373,7 @@ Rails/CompactBlank: # new in 2.13
 Rails/DurationArithmetic: # new in 2.13
   Enabled: true
 Rails/RedundantPresenceValidationOnBelongsTo: # new in 2.13
-  Enabled: true
+  Enabled: false # NOTE: we want this to be false!
```

Comment:
Can you leave a note here about why we want it to be false? "Why" is always the question I wonder about when I look to comments.

Reply:
I switched this back to true and fixed the tests.  In most cases we weren't doing explicit validation in the code, so I only had to change that in one file.